prov/efa/src/rxr/rxr_cq.c

```diff
@@ -711,10 +710,10 @@ void rxr_cq_handle_shm_completion(struct rxr_ep *ep, struct fi_cq_data_entry *cq
 
 static inline
 bool rxr_cq_need_tx_completion(struct rxr_ep *ep,
-			       struct rxr_tx_entry *tx_entry)
```

Comment:
This change is not necessary, because we will keep calling this function with a `tx_entry`.